doc/operator-guide/deployment.rst

```diff
@@ -3,4 +3,36 @@ Deployment
 ###########
 
 * Database
```

Comment:
Use a section header here instead of bullet as shown [here](https://github.com/ralsina/rst-cheatsheet/blob/master/rst-cheatsheet.rst#section-structure).

Reply:
This reference is incredibly helpful, thank you